Makefile

```diff
@@ -1,15 +1,31 @@
+# LESS params
 LESS_DIR = ./static/less
-LESS_FILE = style.less
+LESS_RTL_DIR = ./static/less/rtl
+LESS_TMP_FILE = tmp.less
+
+# CSS params
 CSS_DIR = ./static/css
 CSS_FILE = style.min.css
+CSS_RTL_FILE = style-rtl.min.css
 CSS_TMP_FILE = tmp.css
 
 .PHONY: clean demo build
 
 build: clean
-	lessc $(LESS_DIR)/$(LESS_FILE) > $(CSS_DIR)/$(CSS_TMP_FILE)
+	for f in "`find $(LESS_DIR) -maxdepth 1 -type f`"; do cat $$f >> $(LESS_DIR)/$(LESS_TMP_FILE); done
+	for f in "`find $(LESS_RTL_DIR) -maxdepth 1 -type f`"; do cat $$f >> $(LESS_RTL_DIR)/$(LESS_TMP_FILE); done
```

Comment:
Now, we have just one less file in each of these folders, right?
So, I thing we don't need these two lines.

Reply:
Also, we can use `@include` to add include filles with LESS.

Reply:
The reason I created another subfolder rather than `style-rtl.less` was to give future capability if we needed to add more files. And for the same reason I adjusted build process to loop through folder's content rather than reading _one_ file.

Reply:
Yess `@include` works too and TBH I'm ok with both approach, whatever you feel more comfortable!